tests/k8s_testcases/expected_output/no-strong-equivalence-all-peers-pairs.yaml

```diff
@@ -0,0 +1,336 @@
+query: no_strong_equivalence_print_all_peers_pairs_yaml
+configs:
+- nt_notin
+- nt_notinwithexists
+numerical_result: 1
+textual_result: NetworkPolicy kube-system/allow-ingress-app-notin-predefined is not
+  equivalent in nt_notin and in nt_notinwithexists
+explanation:
+- description: Connections allowed in nt_notin/kube-system/allow-ingress-app-notin-predefined
+    which are different in nt_notinwithexists/kube-system/allow-ingress-app-notin-predefined
+  connections:
+  - src: kube-system/calico-node-mgdlr
+    dst: kube-system/calico-node-ns8kw
+    conn: All connections
+  - src: kube-system/calico-node-mgdlr
+    dst: kube-system/calico-node-ptdgj
+    conn: All connections
+  - src: kube-system/calico-node-mgdlr
+    dst: kube-system/file-plugin-7bfb8b69bf-p86gk
+    conn: All connections
+  - src: kube-system/calico-node-mgdlr
+    dst: kube-system/heapster-7df8cb8c66-zxkk2
+    conn: All connections
+  - src: kube-system/calico-node-mgdlr
+    dst: kube-system/keepalived-watcher-57ghx
+    conn: All connections
```

Comment:
I would avoid having 2 yaml objects for the equivalence query result.
We can have one object, similar to the result of the 2-way-containment query.

Reply:
okay, so I'll resolve this first and after that handle the "_diff_" issue